### PR TITLE
Dynamic page

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
This Microverse project is based on an online website for a conference. I chose a brewery theme for this project. I made a home and about page, following Cindy's design on Behance as much as I could. Changes made :
* Add index.html file
* Add about.html file
* Add home-page.css file
* Add about.css file
* Add README.md file
* Add Linters for HTML, CSS, and JAVASCRIPT
* Add .gitignore file
* Add .github folder
* Add images

Here is a link to a video where I am describing the project: 
https://www.loom.com/share/b4cd1062ed30482fbcccbcfdd93b07a6

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+            <li>
+                Sponsor
+            </li>
+
+            <li>
+                News
+            </li>
+
+            <li>
```
Comment (dasileker) [inline about.html]: - [ ] This tag should be a nav not a section

dnskndzk

File: about.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                My Page
+            </li>
+
+            <li>
+                Logout
+            </li>
+        </ul>
```
Comment (dasileker) [inline about.html:38]: - [ ] The same thing should be for the about page change it to nav please

dnskndzk

File: about.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style/about.css">
+</head>
+<body>
+    <header>
+        <ul>
+            <li>
+                <img src="./images/facebook.svg" alt="an image of a facebook icon">
```
Comment (dasileker) [inline about.html:20]: - [ ] The logo should have an anchor tag so when the user can return back to the home page.

File: style/home-page.css [hunk 4]
```diff
@@ -0,0 +1,770 @@
...
+    flex-direction: column;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    text-align: center;
+    padding: 10px;
+    border-right: transparent;
```
Comment (dasileker) [inline style/home-page.css:621]: - [ ] The CSS in this view is having some issues , please fix in both pages the home page and about page

jdzj

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @hamid-murambiwa,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi @hamid-murambiwa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**